src/gb/gb.c

```diff
@@ -420,6 +420,12 @@ void GBReset(struct LR35902Core* cpu) {
 				memcpy(&gb->memory.romBase[0x100], &gb->memory.rom[0x100], sizeof(struct GBCartridge));
 			}
 
+			if (gb->model == GB_MODEL_AGB) {
+				/* Games can check register B to determine if they are running on GBA */
+				uint8_t ldB1[3] = {0x00, 0x06, 0x01};
```

Comment:
Spaces around the braces (I think, check other parts of my code to see if I'm consistent about this).